website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -0,0 +1,98 @@
+sidebar_current: "docs-azurerm-resource-cosmosdb-account"
+description: |-
+  Creates a new CosmosDB (formally DocumentDB) Account.
+---
+
+# azurerm\_cosmos_db
+
+Creates a new CosmosDB (formally DocumentDB) Account.
+
+## Example Usage
+
+```
+resource "azurerm_resource_group" "test" {
+    name = "resourceGroup1"
+    location = "West Europe"
+}
+
+resource "azurerm_cosmosdb_account" "test" {
+  name                = "cosmosDBAccount1"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  offer_type          = "Standard"
+  consistency_policy {
+    consistency_level = "BoundedStaleness"
+  }
+
+  failover_policy {
+    location = "West Europe"
+    priority = 0
+  }
+
+  failover_policy {
+    location = "East US"
+    priority = 1
+  }
+
+  tags {
+    hello = "world"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the CosmosDB Account. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which the CosmosDB Account is created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `offer_type` - (Required) Specifies the Offer Type to use for this DocumentDB Account - currently this can only be set to `Standard`.
+
+* `consistency_policy` - (Required) Specifies a `consistency_policy` resource, used to define the consistency policy for this DocumentDB account.
+
+* `failover_policy` - (Required) Specifies a `failover_policy` resource, used to define where data should be replicated.
+
+* `ip_range_filter` - (Optional) DocumentDB Firewall Support: This value specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IP's for a given database account. IP addresses/ranges must be comma separated and must not contain any spaces.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+`consistency_policy` supports the following:
+
+* `consistency_level` - (Required) The Consistency Level to use for this CosmosDB Account - can be either `BoundedStaleness`, `Eventual`, `Session` or `Strong`.
+* `max_interval_in_seconds` - (Optional) When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100. Defaults to `5`. Required when `consistency_level` is set to `BoundedStaleness`.
+* `max_staleness` - (Optional) When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated. Accepted range for this value is 1 – 2,147,483,647. Defaults to `100`. Required when `consistency_level` is set to `BoundedStaleness`.
```

Comment:
Some context: the Azure API's won't allow this field to be set to any value unless `consstency_level` is set to `BoundedStaleness`

I'm unsure if that's an API bug, or intentional design decision - that's my only concern with this PR AFAIK